docs/troubleshoot.md

```diff
@@ -6,92 +6,93 @@ indent: true
+This should produce output that includes:
 
 ```console
-kubectl get <resource-type> -o yaml
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
 ```
 
-For example, to get complete information about an Azure AKS cluster object, the following command will generate the below sample (truncated) output:
+Most Crossplane resources set exactly two condition types; `Ready` and `Synced`.
+`Ready` represents the availability of the resource itself - whether it is
+creating, deleting, available, unavailable, binding, etc. `Synced` represents
+the success of the most recent attempt to 'reconcile' the _desired_ state of the
+resource with its _actual_ state.
 
-```console
-> kubectl -n crossplane-system get akscluster -o yaml
-...
-  status:
-    Conditions:
-    - LastTransitionTime: 2018-12-04T08:03:01Z
-      Message: 'failed to start create operation for AKS cluster aks-demo-cluster:
-        containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request:
-        StatusCode=400 -- Please see https://aka.ms/acs-sp-help for more details."'
-      Reason: failed to create cluster
-      Status: "False"
-      Type: Failed
-    - LastTransitionTime: 2018-12-04T08:03:14Z
-      Message: ""
-      Reason: ""
-      Status: "False"
-      Type: Creating
-    - LastTransitionTime: 2018-12-04T09:59:43Z
-      Message: ""
-      Reason: ""
-      Status: "True"
-      Type: Ready
-    bindingPhase: Bound
-    endpoint: crossplane-aks-14af6e93.hcp.centralus.azmk8s.io
-    state: Succeeded
-```
+## Crossplane Logs
+
+The next place to look to get more information or investigate a failure would
+be in the Crossplane pod logs, which should be running in the
+`crossplane-system` namespace. To get the current Crossplane logs, run the
+following:
 
-We can see a few conditions in that AKS cluster's history.
-It first encountered a failure, then it moved into the `Creating` state, then it finally became `Ready` later on.
-Conditions that have `Status: "True"` are currently active, while conditions with `Status: "False"` happened in the past, but are no longer happening currently.
+```shell
+kubectl -n crossplane-system logs -lapp=crossplane
+```
 
 ## Pausing Crossplane
 
-Sometimes, it can be useful to pause Crossplane if you want to stop it from actively attempting to manage your resources, for instance if you have encountered a bug.
-To pause Crossplane without deleting all of its resources, run the following command to simply scale down its deployment:
+Sometimes, it can be useful to pause Crossplane if you want to stop it from
```

Comment:
nitpick: 
could be rephrased as:
Sometimes when you are encountering a bug, it can be useful to pause Crossplane if you want to stop it from actively attempting to manage your resources.